### ISSUE TITLE
Google sign-in problem

### ISSUE THREAD

**Issue by UncleCaptain** [ttalvitie/browservice#84]
I've tried signing into a google account via Browservice 0.95.1, but however, it has given me an error: Couldn't sign you in. This browser or app may not be secure. Try using a different browser. If you're already using a supported browser, you can try again to sign in. Anyone how to fix this problem?

**Comment by UncleCaptain** [comment]
This is the result. I am also running Chrome 49 & Browservice on Windows 2000 (with KernelEx).

**Comment by UncleCaptain** [comment]
![error](https://user-images.githubusercontent.com/111876654/197370146-3fa3daa4-2093-4c2f-bd93-48581eb41ef6.PNG)
This is the result. I am also running Chrome 49 + Browservice on Windows 2000 (with KernelEx.)

**Comment by ttalvitie** [comment]
Yes, unfortunately Google has implemented login restrictions that make it difficult to login with CEF-based browsers. At some point changing the user agent helped, but last time I tried it no longer worked; the detection seems to be on a deeper level. There may still be ways around the block, but I wouldn't rely on them working for a long time, as Google seems quite determined to block CEF-based browsers. I personally won't have time to try to make Browservice work with sites that actively try to block it; it's far outside the intended scope of the project for me.

If you want to read up on the issue, here are a few links (without much in terms of closure or good news):
- Google Developers Blog post about MitM protection (the reason for the block): https://security.googleblog.com/2019/04/better-protection-against-man-in-middle.html
- Additional Google Developers Blog post on the very strict requirements on browsers allowed to log in to Google: https://developers.googleblog.com/2020/08/guidance-for-our-effort-to-block-less-secure-browser-and-apps.html
- An old CEF forum thread about the block, with outdated workarounds: https://magpcss.org/ceforum/viewtopic.php?f=10&t=16717
- A thread started by the main CEF developer bringing up the issue to the Google sign-in team (leads to nowhere): https://groups.google.com/a/chromium.org/g/embedder-dev/c/STyM5ZNTHMM/m/POj1v_cqBgAJ

**Comment by CRUProductions** [comment]
Anyway to fix this problem for Browservice?
It was working earlier.

**Comment by ttalvitie** [comment]
There is a workaround in comment https://github.com/ttalvitie/browservice/issues/113#issue-2060588360 that seems to be currently working; for some unknown reason, setting user agent to "https://accounts.google.com" seems to avoid the block. However, this is not guaranteed to work forever.
